notifiers/common/structs.go

```diff
@@ -0,0 +1,28 @@
+package common
+
+import (
+	"finala/api/storage"
```

Comment:
Do not use finala storage in your notifier command. 
The notifier should be a charge of the response parsing